### PR TITLE
ACR Transfer ARM templates

### PR DESCRIPTION
**Pull request by jaysterp**

### CODE REVIEW

File: docs/image-transfer/ExportPipelines/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+```
+```
+principalID=$(az identity show --resource-group myResourceGroup --name myPipelineId --query principalId --output tsv)
+resourceID=$(az identity show --resource-group myResourceGroup --name myPipelineId --query id --output tsv)
+```
+
+### Grant the identity read access to KV
+
+```
+az keyvault set-policy --name mykeyvault \ --resource-group myResourceGroup \ --object-id $principalID \ --secret-permissions get
```
Comment (sajayantony) [inline docs/image-transfer/ExportPipelines/README.md:22]: Probably need to line break these.

Comment (SteveLasker) [reply docs/image-transfer/ExportPipelines/README.md]: Is this where templates go? I thought Docs either had a location, or the ARM team had a location to post this content? 
@dlepow where do other docs reference templates?

Comment (jaysterp) [reply docs/image-transfer/ExportPipelines/README.md]: @sajayantony I removed the READMEs so we can just link to Dan's docs.

File: docs/image-transfer/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+See public docs:
```
Comment (sajayantony) [inline docs/image-transfer/README.md:1]: You can use https://aks.ms/acr/transfer

**Pull request merged**